### PR TITLE
Feature/style list

### PR DESCRIPTION
**Pull request by katarzynaorlinska**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+    integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
```
Comment (mrruby) [inline public/index.html]: Please use react library

File: src/components/Item/index.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
 import "./item.css";
 
 function Item(props) {
+
+  const numbers = [1, 2, 3, 4, 5];
+  const stars = numbers.map((number) =>
+  <div  className={((number>0)&&(number<=props.stars)) ? 'list__content--active' : 'list__content--off'}><i class="fas fa-star"></i></div>
```
Comment (mrruby) [inline src/components/Item/index.js]: please shorten to 

```
number<=props.stars
```

File: src/components/Item/index.js [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+
+  const numbers = [1, 2, 3, 4, 5];
+  const stars = numbers.map((number) =>
+  <div  className={((number>0)&&(number<=props.stars)) ? 'list__content--active' : 'list__content--off'}><i class="fas fa-star"></i></div>  
+  );
+
   return (
-    <li key={props.id}>
-      {props.name} {props.city}
+    <li key={props.id} className="list">
```
Comment (mrruby) [inline src/components/Item/index.js]: Please change key to something very unique like 

```
`hotel-element-main-list-{props.id}`
```

**Review by mrruby — CHANGES_REQUESTED** [review]

File: src/components/Item/index.js (hunk 2 above)
Comment (katarzynaorlinska) [reply src/components/Item/index.js]: Fixed - shortening the condition

File: src/components/Item/index.js (hunk 3 above)
Comment (katarzynaorlinska) [reply src/components/Item/index.js]: Fixed - renaming key

**Review by mrruby — APPROVED** [review]

**Pull request merged**